### ISSUE TITLE
Bypassing Cloudflare

### ISSUE THREAD

**Issue by cvaclav** [lwthiker/curl-impersonate#218]
Hi,

I would like to ask you about your experience using curl-impersonate to fetch HTML content from site behind Cloudflare.

In fact, it worked well for us at first, but only for a few days. Then the 500 error page started coming back as response. We changed the browser from Chrome to FF, but again, that only worked for a few more days.

Are there any other options or settings to achieve this?

Thanks.

**Comment by saajnn** [comment]
same problem

**Comment by phapntm** [comment]
same problem right now...

**Comment by lux5am** [comment]
Maybe you request too often in short period of time?

**Comment by phapntm** [comment]
> Maybe you request too often in short period of time?

Is that problem? I make many request but using proxy.

**Comment by lux5am** [comment]
> > Maybe you request too often in short period of time?
> 
> Is that problem? I make many request but using proxy.

yes. common problem. especially if you use proxy where it can be detected. the ip would be flagged as a bot. Some website uses stronger couldflare security setting. cloudflare wrap, nordvpn and other popular VPN already marked. just be moderate. use random sleep here and there

**Comment by ukd1** [comment]
checkout [FlareSolverr/FlareSolverr](https://github.com/FlareSolverr/FlareSolverr) for that kind of thing
